composer.json

```diff
@@ -0,0 +1,42 @@
+{
+    "name": "laravel-notification-channels/infobip",
+    "description": "Infobip notification channel for Laravel",
+    "homepage": "https://github.com/laravel-notification-channels/infobip",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Thomson Maguru",
+            "email": "cliff@magurutechnologies.com",
+            "homepage": "httsp://magurutechnologies.com",
+            "role": "Developer"
+        }
+    ],
+    "require": {
+        "php": ">=7.2",
+        "illuminate/notifications": "~5.5 || ~6.0 || ~7.0",
+        "illuminate/support": "~5.5 || ~6.0 || ~7.0",
+        "infobip/infobip-api-php-client": "dev-master"
```

Comment:
Could we use a tagged release here?

Reply:
Unfortunately, they only have that in their API. Its been so for quite some time now. But it works well and they do maintain it. [Check it Here](https://github.com/infobip/infobip-api-php-client)